docs/app/Examples/views/Statistic/Types/StatisticGroupExample.js

```diff
@@ -0,0 +1,24 @@
+import React, {Component} from 'react';
+import {Statistic} from 'stardust';
+const {Statistics, Label, Value} = Statistic;
```

Comment:
This is a bit overwhelming to look at. I think I would rather see `<Statistic value="22" label="Faves" />` but I realize the `.Value`, etc provides good options